packages/sdks/snippets/react/src/routes/targeted-page/setUserAttributes.tsx

```diff
@@ -0,0 +1,8 @@
+import { setClientUserAttributes } from '@builder.io/sdk-react';
+
+export default function setTargetingAttributes() {
+  setClientUserAttributes({
+    device: 'desktop',
+    audience: ['recent-shopper', 'womens-fashion'],
+  });
+}
```

Comment:
@samijaber Here is the option that's failing. I am using this function and file similarly in the Gen 1 directory.

This function _does_ set the cookies appropriately but `fetchOneEntry()` does not seem to pick up the details in the cookie.

Reply:
@samijaber I was able to get this passing with a change to the content entry rather than a change to the code.